### DISCUSSION TITLE
Thoughts for Dev Call - 16 Apr 2023

### DISCUSSION THREAD

**Discussion by richb-hanover** [PRQL/prql#2461] [General]
@max-sixty Regrettably, I will miss the Dev Call today. Here are the questions I would ask:

- Review of questions in the Expressions and Operators page in #2460
- Rebuild VS Code extension to incorporate the 0.8.0 language changes
- I have a question about thoughts for _modules_. Will this be a simple `#include` facility that simply inserts the text of the included file at the point in the PRQL file? Or will there be some deeper understanding of what _modules_ might entail. (I could live with the former. Links to any proposals (half-baked or not) would be appreciated.

Thanks!

**Comment by max-sixty** [comment]
Thanks!

> Rebuild VS Code extension to incorporate the 0.8.0 language changes

Great — dependabot will come along and bump it soon; if anyone wants it earlier then PRs are welcome...

>     * I have a question about thoughts for _modules_. 

Still very active discussion at https://github.com/PRQL/prql/pull/2129 on how this works. There's lots there, no super easy to engage with at this point, but also difficult to compress given there is actually a lot there. https://github.com/PRQL/prql/pull/2129#discussion_r1163243612 is probably the most relevant thread. Productively engaging does require understanding the tradeoffs of the current proposals
